cluster/examples/kubernetes/ceph/monitoring/prometheus-ceph-v14-rules.yaml

```diff
@@ -184,6 +184,34 @@ spec:
       for: 1h
       labels:
         severity: warning
+  - name: persistent-volume-alert.rules
+    rules:
+    - alert: PersistentVolumeUsageNearFull
+      annotations:
+        description: PVC {{ $labels.persistentvolumeclaim }} utilization has crossed
+          75%. Free up some space or expand the PVC.
+        message: PVC {{ $labels.persistentvolumeclaim }} is nearing full. Data deletion
+          or cluster expansion is required.
```

Comment:
PVC expansion or cluster expansion?

Reply:
Thanks @Madhu-1 I missed that. I will update